### ISSUE TITLE
Chapter 16 (Predicting DFT Energies with GNNs) Feedback

### ISSUE THREAD

**Issue by vjain31** [whitead/dmol-book#55]
1. In table 16.1 (Label Description), for index 7's description it should be :Energy of Lowest **unoccupied** molecular orbital (LUMO). Currently it's **occupied**.
2. you used at multiple places "it's, we'll, I'll, we've etc.", I personally don't like in the books this language, I would suggest to use it is, we will, I will, we have etc.
3. On page 2 of chapter 16, in the paragraph "These are Tensorflow Tensors. They can be converted to numpy arrays via x.numpy(). The first item is the element vector 6,1,1,1,1. Do you recognize the elements? It’s C, H, H, H, H. The positions come next. Note that the there is an extra column containing the atom partial charges, which we will not use as a feature. Finally, the last tensor is the label vector.", there is an extra "the" in the sentence "Note that **the** there is an extra column containing the atom partial charges, which we will not use as a feature. Finally, the last tensor is the label vector."
4. For each plot, you should add units for 'X' and 'Y' axis fields such as energies for Val Loss (if any), Epoch (if any), Energy (if any) and predicted Energy (if any).
5. There should be "a comma (,)" after "thus" in the sentence " Thus we start at high learning rate and decrease." in the paragraph "This is poor performance, but it gives us a baseline value of what we can expect. One unusual detail I did in this training was to slowly reduce the learning rate. This is because our features and labels are all in different magnitudes. Our weights need to move far to get into the right order of magnitude and then need to fine-tune a little. Thus we start at high learning rate and decrease."
6. It should be "variances" in the sentence "You can see that the broad trends about molecule size capture a lot of **variance**, but more work needs to be done."
7. There should be "a comma (,)" after "finally" in the sentence "The global node aggregation will also be a sum. **Finally** we have our graph feature vector update:"

**Comment by whitead** [comment]
Hi @vjain31,

Thanks for all this effort to prepare such detailed feedback! I will incorporate your feedback into a revision for the chapter.

**Comment by whitead** [comment]
Fixed by 5c71185. 

@vjain31 Note that some of your stylistic changes are a matter of opinion, like using it's vs it is. I'll stick to my own voice since that is how the rest of the book is written.
